src/components/input-panel/compiled-spec-display/index.js

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
-import AceEditor from 'react-ace';
 import ReactResizeDetector from 'react-resize-detector';
 import { MODES, LAYOUT } from '../../../constants';
 import { connect } from 'react-redux';
+import MonacoEditor from 'react-monaco-editor';
 
-import 'brace/mode/json';
-import 'brace/theme/github';
+// import 'brace/mode/json';
+// import 'brace/theme/github';
+
+  const toggleStyle = {
+    position: 'absolute',
+    bottom: '0px',
+    color: 'white',
+    cursor: 'pointer',
+    display: 'inline-block',
+    fontSize: '15px',
+    width: '100%',
+    height: '20px',
```

Comment:
When defining pixel values on a react style object, just use a number and omit `px`, so this becomes: `height: 20`